src/widgets/WebexMeeting/README.md

```diff
@@ -17,9 +17,16 @@ To get a preview of the Webex Meeting widget, you can run our start script:
 ## Embed
 
 Import the Meeting widget from the `@webex/widgets` library and embed it in your React application
+by passing it the access token as a string, the meeting destination as a string and an optional function. This function returns an array 
+of control names for the meeting. The default control names are set to `['mute-audio', 'mute-video', 'settings', 'join-meeting']` if the meeting is inactive and `['mute-audio', 'mute-video', 'share-screen', 'member-roster', 'settings', 'leave-meeting']`otherwise.
```

Comment:
```suggestion
The default control names are set to `['mute-audio', 'mute-video', 'settings', 'join-meeting']` if the meeting is inactive and `['mute-audio', 'mute-video', 'share-screen', 'member-roster', 'settings', 'leave-meeting']`otherwise.
```